src/api/getCurrentUser.ts

```diff
@@ -0,0 +1,8 @@
+export const getCurrentUser = (token: string) =>
+  fetch("/api/me", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({ token }),
```

Comment:
What would happen if you would do: 
```suggestion
    body: token,
```
?

Reply:
It does not work properly, in th case of JSON.stringify({ token }) Im sending:

{
  "token": "abc123"
}

and in the case of "token", Im sending "abc123" apparently, and getting 400